### DISCUSSION TITLE
RX-V673 works

### DISCUSSION THREAD

**Discussion by Maco65** [mvdwetering/yamaha_ynca#266] [General]
Thank you. Works very well ans stable.
![image](https://github.com/mvdwetering/yamaha_ynca/assets/18126641/5eb631cf-c2b1-4477-a427-9af1b01bca84)

My challenge is now to see if and how it is possible to use it as media player rfom Spotify Add-on, or to play internet radion from HA on amplifire (not using built in internet radion from Yamaha). Trying to read and find some answers... :)

**Comment by mvdwetering** [comment]
The protocol I am using for this integration does not offer a way to playback audio from a URL or something.

You might want to try the "DLNA Digital Media Renderer" that comes with Home Assistant. It supports playing back audio, but audio has to be supported by the receiver.  For my rather old receiver it does not work very well. I assume Yamaha improved it in later models.

**Comment by Maco65** [comment]
Thank you.
I have managed to setup DLNA Media Render - it looks OK however stability might be a challenger. Will be watching it.
THANKS
